tekton/ci/jobs.yaml

```diff
@@ -65,3 +65,26 @@ spec:
       echo "Content of the repo:"
       ls $(resources.inputs.source.path)
       exit $(( $(date +"%s") % 2 ))
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: yamllint
+  namespace: tektonci
+  description: |
+    Lint yamls
+spec:
+  resources:
+    inputs:
+      - name: source
+        type: git
+    outputs:
+      - name: endtrigger
+        type: cloudEvent
+  steps:
+  - name: lint
+    image: gcr.io/tekton-releases/dogfooding/yamllint:latest
+    script: |
+      #!/bin/sh
+      set -ex
+      yamllint .
```

Comment:
So, is this running on the whole repo?
Shall we have white/black list of folders? 
I don't want to yamllint vendor?

Reply:
yeah :stuck_out_tongue_closed_eyes: we can refine that command indeed :stuck_out_tongue_closed_eyes: